### ISSUE TITLE
Crash when LineChart is re-rendered

### ISSUE THREAD

**Issue by bosung90** [indiespirit/react-native-chart-kit#45]
The app crashes when LineChart is re-rendered even without any changes to the graph (data, width, height, chartConfig are all same)

iOS crash 
`removedChildren count (0) was not what we expected (1)`
Android crash

```
Trying to remove a view index above child count 2 view tag: 403
detail: View tag:403
 children(0): [
],
 indicesToRemove(1): [
2,
],
 tagsToDelete(1): [
307,
]
```

versions:
    "react": "16.3.1",
    "react-native": "^0.56.0",
    "react-native-chart-kit": "^1.1.6",
    "react-native-svg": "^7.0.2",

reproducible code

```
import React from 'react'
import { LineChart } from 'react-native-chart-kit'

const data = {
  labels: ['January', 'February', 'March', 'April', 'May', 'June'],
  datasets: [
    {
      data: [20, 45, 28, 80, 99, 43],
    },
  ],
}

export default class DailyRevenue extends React.Component {
  componentDidMount() {
    setTimeout(() => {
      this.setState({})
    }, 1000)
  }
  render() {
    return (
      <LineChart
        data={data}
        width={400}
        height={400}
        chartConfig={{
          backgroundColor: '#e26a00',
          backgroundGradientFrom: '#fb8c00',
          backgroundGradientTo: '#ffa726',
          color: (opacity = 1) => `rgba(255, 255, 255, ${opacity})`,
          style: {
            borderRadius: 16,
          },
        }}
      />
    )
  }
}
```

**Comment by bosung90** [comment]
I just tried with "react-native-svg": "^6.4.1"
No longer crahses

**Comment by tarunmehta-quovantis** [comment]
I am facing the same issue but is not resolved even after changing version to 6.4.1
any other solution?
@bosung90

**Comment by cristian1703** [comment]
for me "react-native-svg": "6.3.1" works
